packages/client/src/views/GrantDetailsView.vue

```diff
@@ -350,6 +367,12 @@ export default {
     statusSubmitButtonDisabled() {
       return this.selectedInterestedCode === null;
     },
+    displayEstimatedCloseDateText() {
+      return this.currentGrant.close_date && this.currentGrant.opportunity_status === 'forecasted';
+    },
+    displayEstimatedOpenDateText() {
+      return this.currentGrant.open_date && this.currentGrant.opportunity_status === 'forecasted';
+    },
```

Comment:
Note to reviewer:  We do not want to display prefix "est." if the dates are null and the status is forecasted.

valid date - "est. 1/2/2056"
date = null - "not yet issue"

We need this as a separate flag because the requirement is to display "est." in italics.